kayvee.go

```diff
@@ -19,7 +20,11 @@ const (
 
 // Format converts a map to a string of space-delimited key=val pairs
 func Format(data map[string]interface{}) string {
-	formattedString, _ := json.Marshal(data)
+	formattedString, err := json.Marshal(data)
+	if err != nil {
+		log.Printf("Error converting kayvee message to json: %s", err.Error())
```

Comment:
It would be consistent :) 

Not picky here though. If logged as Kayvee, we'd want to make sure the source is specified as `kayvee-go` to ensure its not confused with an application issue.